src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -579,6 +593,71 @@ public void onFocusCenterSeparator()
       panel_.focusSplitter();
    }
 
+   @Handler
+   public void onFocusNextPane()
```

Comment:
Several tab types in the Source panel don't yet handle focus and prevent F6 from moving to that panel when they are selected.

- profile results (`ProfilerEditingTarget`)
- `UrlContentEditingTarget` (execute `data()` in the console)
- `DataEditingTarget` (execute `View(mtcars)` in console)
- `ObjectExplorerEditingTarget` (execute `View(list("a" = 1.2, "b" = TRUE, "c" = 3:10))` in console)

Fine with you opening a separate issue on this and looking into it later.